# from django.core.files.storage import default_storage
# from django.core.exceptions import ValidationError
#
# def user_image_save_with_folder_name(image):
#
#     if default_storage.exists(f'user/profile_images/{image.user.username}'):
#         pass
#     else default_storage.exists(f'user/profile_images/{image.user.username}'):
#         pass